AsyncDisplayKit/ASCellNode+Internal.h

```diff
@@ -29,6 +29,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)nodeDidRelayout:(ASCellNode *)node sizeChanged:(BOOL)sizeChanged;
 
+/// Returns the constrained size for the given node
+- (ASSizeRange)constrainedSizeForNode:(ASCellNode *)node;
```

Comment:
Yes agree, I added it explicitly for a use case we don't need anymore as we pass in the new size in `displayNodeDidInvalidateSizeNewSize:`